bloom/generators/debian/templates/cmake/rules.em

```diff
@@ -59,11 +59,24 @@ override_dh_shlibdeps:
 	# in the install tree and source it.  It will set things like
 	# CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/:$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/${DEB_HOST_MULTIARCH}
+@[if RuntimePackage]
+	dh_shlibdeps -l$(CURDIR)/debian/@(Package)-runtime@(InstallationPrefix)/lib/:$(CURDIR)/debian/@(Package)-runtime@(InstallationPrefix)/lib/${DEB_HOST_MULTIARCH}
+@[else]
+	dh_shlibdeps -l$(CURDIR)/debian/@(Package)-runtime@(InstallationPrefix)/lib/:$(CURDIR)/debian/@(Package)-runtime@(InstallationPrefix)/lib/${DEB_HOST_MULTIARCH}
```

Comment:
I guess the -runtime should not be here in this brach...

Reply:
fixed, thanks!